app_server/views/service.pug

```diff
@@ -1,43 +1,215 @@
 extends navbarCentre
 
 block content
-    
+
     include _includes/submitModal
-    
-    #banner.page-header
+
+    .container-fluid
+      h2 Service Dashboard
+      .row
+        .col-xs-12
+          .panel.panel-default
+            .panel-body
+              .col-sm-9
+                h4 Youth Accommodation Support Service
+                h5 Victoria Park
+              .col-sm-3
+                .pull-right
+                  h4
+                    | Available Beds:
+                    span.badge.badge-success 7
```

Comment:
Add some space here because there is no space `Available Beds:7`
You can do this in many ways preferably with some existing classes by bootstrap of simply add a space character or styles e.g. `padding-right:5px;`